common/logger/src/struct_log.rs

```diff
@@ -38,30 +38,40 @@ const INITIALIZED: usize = 2;
 // severity level - lower is worse
 const SEVERITY_CRITICAL: usize = 1;
 const SEVERITY_WARNING: usize = 2;
+const SEVERITY_SECURITY: usize = 3;
```

Comment:
I added security logs as a severity level after discussion with @andll

Reply:
// severity level - lower is worse
To keep this ordering, should we rearrange those?
SEVERITY_SECURITY -> 1
SEVERITY_CRITICAL -> 2
SEVERITY_WARNING -> 3